slack_cleaner/cli.py

```diff
@@ -187,6 +189,18 @@ def get_direct_id_by_name(name):
                 return i['id']
 
 
+def get_mpdirect_id_by_name(name):
+    res = slack.mpim.list().body
+    name = 'mpdm-' + name + '-1'
```

Comment:
So it makes `--mpdirect sherry,james` different from `--mpdirect james,sherry`.  Another approach I can think of is finding the group id by matching `set(names.split(","))` and `set(res['groups']['members'])`.  Also, the naming policy `'mpdm-' + names + '-1'` seems more likely to change over time than `res['groups']['members']`.